### ISSUE TITLE
Debug messages on console when registered

### ISSUE THREAD

**Issue by blueglyph** [kwhat/jnativehook#307]
In release 2.1.0 (of 3 years ago), there are a lot of INFO logs on the console as soon as the GlobalScreen.registerNativeHook() is called.

Java 1.8 is used, on Windows (64-bit), Java 11.0.8 was also tested without any difference.

Is it possible that in the 2.1.0 release, ENABLE_DEBUG:BOOL has been accidentally left to TRUE when compiling uiohook?

Excerpt of those messages:

> Sep 13, 2020 12:47:50 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_mouse_moved [459]: Mouse moved to 444, 564.
> 
> Sep 13, 2020 12:47:50 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_mouse_moved [459]: Mouse moved to 444, 563.
> 
> Sep 13, 2020 12:47:50 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_mouse_moved [459]: Mouse moved to 444, 562.
> 
> Sep 13, 2020 12:47:50 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_mouse_moved [459]: Mouse moved to 443, 562.
> 
> Sep 13, 2020 12:47:50 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_mouse_moved [459]: Mouse moved to 443, 562.
> 
> Sep 13, 2020 12:47:51 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_key_pressed [227]: Key 0X1 pressed. (0X1B)
> 
> Sep 13, 2020 12:47:51 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: refresh_locale_list [470]: GetKeyboardLayoutList(0, NULL) found 2 layouts.
> 

The hook is registered like so, the class GlobalKeyListenerExample only dumps some information and is not the cause for this issue:

```
		try {
			GlobalScreen.registerNativeHook();
		} catch (NativeHookException ex) {
			System.err.println("There was a problem registering the native hook.");
			System.err.println(ex.getMessage());

			System.exit(1);
		}

		//Construct the example object and initialze native hook.
		GlobalScreen.addNativeKeyListener(new GlobalKeyListenerExample());
```

Interestingly, there are recurrent messages like those, which is strange:

> Sep 13, 2020 12:47:51 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> SEVERE: refresh_locale_list [628]: Could not find keyboard map for locale 0XFFFFFFFFF0C0080C!
> 
> Sep 13, 2020 12:47:51 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> SEVERE: refresh_locale_list [628]: Could not find keyboard map for locale 0XFFFFFFFFF0C1080C!

And also those, not related to keyboard:

> Sep 13, 2020 12:58:35 PM org.jnativehook.GlobalScreen$NativeHookThread enable
> INFO: process_mouse_moved [459]: Mouse moved to 746, 903.

**Comment by kwhat** [comment]
Its fixed in 2.2, there should be a snapshot in maven.  No official release yet.

**Comment by szw19** [comment]
There's  part of the source code for class GlobalScreen:
![image](https://user-images.githubusercontent.com/61656369/93707714-977b5d80-fb63-11ea-862d-7dbea5c97578.png)
It use class Logger(in JDK)  to print log. So you can close turn off print by setting the log level specifying  of class  
 GlobalScreen,just like this:
![image](https://user-images.githubusercontent.com/61656369/93707859-8da62a00-fb64-11ea-8911-e112c960e4a9.png)

**Comment by kwhat** [comment]
This should be resolved in 2.2
